.github/workflows/gh-pages.yml

```diff
@@ -0,0 +1,25 @@
+---
+name: Sphinx build
+
+on: push
```

Comment:
I don't think this is quite what we want.  The published docs should correspond to released versions, not every push on every branch.

Reply:
This is maybe something, we want to discuss.
Atm, I have `if: github.ref == 'refs/heads/main'` for the publish step. This should publish only when pushing to main.
The intention was to have "dev" docs.
Here, we also might consider readthedocs, as this would allow us to have different versions of the docs (one for each release, plus one dev version for example).
But maybe this is also overkill and we actually only want to have docs for the latest release.

Reply:
Resolving this one, as there is a new thread below